deploy/toolchaincluster/member-sa.yaml

```
@@ -0,0 +1,109 @@
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: toolchaincluster-member
+  namespace: {{.Namespace}}
+rules:
+- apiGroups:
+  - toolchain.dev.openshift.com
+  resources:
+  - "idlers"
+  - "nstemplatesets"
+  - "memberoperatorconfigs"
+  - "memberstatuses"
+  - "toolchainclusters"
+  - "useraccounts"
+  verbs:
+  - "*"
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: toolchaincluster-member-{{.Namespace}}-toolchaincluster
```

I know that we have this crazy long names created by the script, but we could maybe make it shorter and clean the old ones manually. It shouldn't have any impact on the functionality of the controllers - it's just ClusterRole & ClusterRoleBinding, we don't change anything in the ServiceAccount.
```suggestion
  name: toolchaincluster-{{.Namespace}}
```
WDYT?

---

makes sense, I didn't want to change those names initially. 

Done in https://github.com/codeready-toolchain/member-operator/pull/537/commits/5b6bdb04ebcbfbf5b1c1648dc1b7c7e768def36b

Thanks!